registry/registry.go

```diff
@@ -99,7 +101,12 @@ func doRequest(req *http.Request, jar http.CookieJar, timeout TimeoutType, secur
 			return false
 		}
 
-		hostDir := path.Join("/etc/docker/certs.d", req.URL.Host)
+		if runtime.GOOS == "windows" {
+			hostDir = filepath.Join(os.TempDir(), "/docker/certs.d")
```

Comment:
oh wow I wasn't aware of the registry pkg. In any case, I don't think continue using registry, files are much easier to work with. (unix philosophy). :smile: